pages/web/hosting/exporter-son-site-web/guide.en-asia.md

```diff
@@ -62,6 +62,8 @@ To identify the folder that the website is stored on, go to the `Multi-site`{.ac
 To retrieve a backup of your database, please read our guide on
 [Retrieving the backup of a Web Hosting plan’s database](../web_hosting_database_export_guide/).
 
+If you are using a **CloudDB** database for your website, you can go to the ‘Databases’ tab, click the `...`{.action} button to the right of the database, and click ‘Back up now’ in the drop-down menu.
```

Comment:
@Kazarus4 , CloudDB is not offered outside Europe. Please remove it from all CA/APAC subs.

Reply:
done